guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc

```diff
@@ -18,7 +18,8 @@ Installing GSS-proxy and {nfs-client-package}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} sssd adcli realmd ipa-python-compat krb5-workstation samba-common-tools
+# {project-package-install} adcli ipa-python-compat krb5-workstation \
```

Comment:
I wonder why we have prerequisites in this procedure to install some packages, but then install some more packages. Shouldn't that be a single command?

Also, why do we need `nfs-utils` for this? It was added in 9c4054406c9076db5abb2447dd09f4d4b8f135a6 but conceptually I don't see why it would be needed.

Also, on EL9 there is no `ipa-python-compat` package.

Reply:
You make a good point. If possible, I'd like to drop this for now until we review the whole user story.